lib/build/README.md

```diff
@@ -0,0 +1,21 @@
+# Symbolica Build Image
+
+## Building the image
+
+From this directory run the following command:
+
+```sh
+docker build . -t symbolica/build:latest
+```
+
+## Running the image
+
+```sh
+docker run -v $(pwd)/<path-to-user-code>:/code ssvm-build:latest </path/to/user/build.sh>
```

Comment:
This should say `symbolica/build:latest` instead of `ssvm-build:latest`. And in the example below please change it from `/tests/pass/c/average` to just `mycode` or `./mycode` or something (because the leading `/` implies root and is slighly confusing).

And maybe mention that `</path/to/user/build.sh>` is optional if you like, but I don't mind either way really.

Apart from that, LGTM, where G == Glorious.